controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/systemflags/v1/wire/SystemFlagsV1Api.java

```diff
@@ -20,6 +18,12 @@
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes("application/zip")
     @Path("/deploy")
-    WireSystemFlagsDeployResult deploy(@QueryParam("dryRun") @DefaultValue("false") boolean dryRun, InputStream inputStream);
+    WireSystemFlagsDeployResult deploy(InputStream inputStream);
+
+    @PUT
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes("application/zip")
+    @Path("/dryrun")
+    WireSystemFlagsDeployResult dryrun(InputStream inputStream);
```

Comment:
"dryrun" is not very descriptive compared to "deploy", especially if /system-flags/v1 gets more path bindings. Should probably be called "deployprobe" or something?